### PR TITLE
Update master to V2

### PR DESCRIPTION
**Pull request by bobylito**
Build scripts have changed so we need to merge to master first :)

### CODE REVIEW

File: docgen/assets/js/api.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/* eslint-disable no-console */
+
+console.log('api');
```
Comment (Haroenv) [inline docgen/assets/js/api.js]: What's this for?

**Comment by Haroenv** [comment]
Travis PR fails because it's done to master: `Error: Pull request cannot be done on master`

[link](https://travis-ci.org/algolia/instantsearch.js/builds/248683716?utm_source=github_status&utm_medium=notification)

File: docgen/assets/js/api.js (hunk 1 above)
Comment (bobylito) [inline docgen/assets/js/api.js]: tests :/ should remove that I guess.

**Comment by bobylito** [comment]
> Travis PR fails because it's done to master: Error: Pull request cannot be done on master

We'll have to merge manually then :/

File: docgen/assets/js/api.js (hunk 1 above)
Comment (bobylito) [inline docgen/assets/js/api.js]: Not even test. Wtf is it there? 😓

**Comment by Haroenv** [comment]
We can still merge here, why would it need to be manual @bobylito

**Comment by bobylito** [comment]
> We can still merge here, why would it need to be manual @bobylito

Because I thought it was illegal :)

**Comment by bobylito** [comment]
> I guess this is good, but it's hard to review 😄

TBH this is impossible to review completely. Everything was already reviewed. Just check that we are on script side of things. Thanks for the time :)

**Comment by Mark-Howe** [comment]
Possibly wrong place to report, but this broke the links on https://community.algolia.com/instantsearch.js/examples/
https://github.com/algolia/instantsearch.js/tree/master/docs/examples/tourism
needs changing to https://github.com/algolia/instantsearch.js/tree/master/docgen/src/examples/tourism etc

**Comment by julienpa** [comment]
Congrats guys! Looking forward to using it 😃

**Comment by bobylito** [comment]
@Mark-Howe I don't see any errors. The new website lives [on /v2](https://community.algolia.com/instantsearch.js/v2/) and we made the mistake to use the name of the former doc source folder for the new destination folder.

Let us know if you find any broken links :)

**Pull request merged**